tests/unit/test_global_config_update.py

```diff
@@ -207,3 +208,31 @@ def test_config_validation_when_placeholder_is_present(tmp_path, caplog):
     expected_schema_version = "0.0.7"
     assert expected_schema_version == global_config.schema_version
     assert error_log in caplog.text
+
+
+def test_dump_enable_from_global_config_enable_present(tmp_path, caplog):
```

Comment:
Let's add this as a quick separate PR, I'd happily merge it and then we can look at this one.

Reply:
Removed the test cases and raised a separate PR #1454